### ISSUE TITLE
(Win) GL1 driver issues

### ISSUE THREAD

**Issue by eugene-s-nesdev** [libretro/RetroArch#11333]
Hardware:
- Retroarch 1.9.0 x64 windows build
- Asus eeePC 1015PEM, Win7x64, intel GMA 3150 (driver ver 8.14.10.2230), 1024x600.
- Only OpenGL1 hardware support

Bug description:
- Start retroarch windowed at 2x scaler, then start any core/any game
- You'll see half-picture is hidden at bottom screen
- Go to fullscreen, then go back windowed - in both cases picture will use 1x scaler.

verbose and config in attach
![gl1_coldstart](https://user-images.githubusercontent.com/13199529/93121598-c742e500-f6cd-11ea-85ab-53c58eda3bce.png)

![gl1_after_exit_fullscreen](https://user-images.githubusercontent.com/13199529/93121596-c6aa4e80-f6cd-11ea-9120-192232898b31.png)

[verbose.zip](https://github.com/libretro/RetroArch/files/5220152/verbose.zip)

**Comment by eugene-s-nesdev** [comment]
If you set: Settings -> On-ScreenDisplay -> On-ScreenNotifications -> Graphic Widgets to "OFF"
this bug always happened. (i tested on Nvidia ION2 also)

When Graphic Widgets set to "ON" - bug partially gone. Window scales goes normal after returning fullscreen.

**Comment by eugene-s-nesdev** [comment]
I did tests on clean config. Root of the issue was: 
fps_show = "true"

and it affects GL1 driver
[bug_config.zip](https://github.com/libretro/RetroArch/files/5223255/bug_config.zip)

**Comment by i30817** [comment]
Did you try with the notification style without widgets?

settings->On Screen Display->On screen notifications->Graphics Widgets (and settings around there?)

I suppose even if that fixes it making certain settings inoperative at runtime with only certain drivers is harder than it looks.

**Comment by eugene-s-nesdev** [comment]
Yes, i tried notification style with disabled graphic widget. You can try my attached configs.

I did more deeper tests on clean config at different machines (intel hd graphics, nvidia)
and now roots of the issue even more clear:

- it happens only when used GL1 driver
- it happens ONLY when graphical widgets DISABLED
- it triggered by any text, not only FPS-meter. In my last case it was bluetooth-gamepad notification.

Try to represent this.

**Comment by i30817** [comment]
There was this suspicious 'cleanup' commit very recently (6 hours ago):

https://github.com/libretro/RetroArch/commit/a49b81c161f6b6c86476c6a4903c6dc1bb26fbb5#diff-446050614d6936276351afb8a46e2d10L487

Notice how the font and width are switched. Might be the solution if you can build.

**Comment by eugene-s-nesdev** [comment]
Sorry, i can't build now, i will wait nightly builds.
But when i did these tests, i've found another bug like this

Try to repeat:
- it happens only when used GL1 driver
- it happens independently of graphical widgets enabled or disabled

Set:
- windowed mode, 2x for example
- settings - video - scaling - aspect ratio - custom
`Custom Aspect Ratio (X Position) = 0
Custom Aspect Ratio (Y Position) = 0
Custom Aspect Ratio (Width) = 640
Custom Aspect Ratio (Height) = 480
Crop Overscan = ON
`
- run ROM

You'll see half-picture is hidden at bottom screen, like my first screenshot.
Enter fullscreen (F), then go back windowed - bug is gone.

**Comment by i30817** [comment]
That deleted post was misinfo, i didn't read it carefully enough.
